i18n/transifex.py

```diff
@@ -118,7 +117,10 @@ def clean_locale(configuration, locale):
     Iterates over machine-generated files.
     """
     dirname = configuration.get_messages_dir(locale)
-    for filename in ('django-partial.po', 'djangojs-partial.po', 'mako.po'):
+    if not dirname.exists():
+        # Happens when we have a supported locale that doesn't exist in Transifex
+        return
+    for filename in dirname.files('*.po'):
```

Comment:
I figure it made sense to just change everything in the folder. The files we pull are probably named django.po and djangojs.po rather than partials. And some consumers won't have mako, etc.

Just clean everything we find. We "own" it all, right?

Reply:
Agree.